envi/archs/arm/disasm.py

```diff
@@ -1881,6 +1899,59 @@ def repr(self, op):
             retval.append(shift_names[self.shtype])
         return " ".join(retval)
 
+class ArmRegShiftXtndOper(ArmOperand):
```

Comment:
See top comment for class.
ShiftImmOper has exact value to shift in the instruction. (Example such as ROR A8.8.149)
ShiftXtndOper can only rotate by 0, 8, 16 or 32 depending on bits 10 and 11 which are the selector bits. The Xtend name comes from the fact these are all sign extend commands.
If you look at SXTAB (A8.8.230 in ARMV7 reference) it shows an example.